crates/smartmodule-development-kit/src/test.rs

```diff
@@ -165,6 +165,11 @@ impl TestCmd {
 
         let output = chain.process(sm_input, &metrics)?;
 
+        if let Some(error) = output.error {
+            print!("SmartModule error: {error}");
+            return Ok(());
```

Comment:
It seems reasonable to return early but I am not aware if it possible to have successful records along side errors in the same output.